packages/appstore/src/components/modals/real-wallets-upgrade/components/content-with-link/content-with-link.scss

```
@@ -0,0 +1,94 @@
+.content-with-link {
+    height: auto;
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: center;
+
+    @include mobile {
+        flex-direction: column-reverse;
+    }
+
+    &--has-fork {
+        height: 100%;
+    }
+
+    &--right-forked {
+        flex-direction: row;
+
+        @include mobile {
+            flex-direction: column;
+        }
+    }
+
+    &__content {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
+        gap: 1.6rem;
+        margin: 0rem;
+        margin-left: 1.6rem;
+
+        @include mobile {
+            gap: 0.8rem;
+            margin: 0;
+            margin-top: 1.2rem;
+        }
+
+        &--right-forked {
+            margin: 0rem;
+            margin-right: 1.6rem;
+
+            @include mobile {
+                margin: 0rem;
+                margin-bottom: 1.2rem;
+            }
+        }
```

Would look more consistent if all margins are written as 0 instead of 0rem
```suggestion
        @include mobile {
            gap: 0.8rem;
            margin: 0;
            margin-top: 1.2rem;
        }

        &--right-forked {
            margin: 0;
            margin-right: 1.6rem;

            @include mobile {
                margin: 0;
                margin-bottom: 1.2rem;
            }
        }
```

---

Hi @lubega-deriv, yes I missed on these 😅. Thanks for pointing out 😄.